sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/AccessControlChangeFailure.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+/**
+ * Represents an entry that failed to update Access Control List.
+ */
+public class AccessControlChangeFailure {
+    private String name;
+    private boolean isDirectory;
+    public String errorMessage;
+
+    /**
+     * Returns the name of an entry.
+     *
+     * @return The name of an entry.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Sets the name of an entry.
+     *
+     * @param name The name of an entry.
+     * @return The updated object.
+     */
+    public AccessControlChangeFailure setName(String name) {
+        this.name = name;
+        return this;
+    }
+
+    /**
+     * Returns whether entry is a directory.
+     *
+     * @return Whether the entry is a directory.
+     */
+    public boolean isDirectory() {
+        return isDirectory;
+    }
+
+    /**
+     * Sets whether entry is a directory.
+     *
+     * @param directory Whether entry is a directory.
+     * @return The updated object.
+     */
+    public AccessControlChangeFailure setDirectory(boolean directory) {
```

Comment:
This sounds kinda weird. Do we have other examples of this? It feels more normal to call this setIsDirectory

Reply:
I agree. But can we do setIsDirectory and isDirectory. I'm not sure if the guidelines ok with there being an extra "is" in the setter.